scripts/build_docs.py

```diff
@@ -28,6 +28,10 @@
 
 def build_docs(versions, use_yarn):
 
+    # define the folders that Docusaurus builds into and a temporary one
```

Comment:
I just moved this to the top of the function, and gave it more descriptive names. We didn't need to reassign the variable at each iteration of the for loop.